docs/user/integration/dynatrace/README.md

```diff
@@ -137,14 +136,20 @@ To start ingesting custom spans and Istio spans, you must enable the Istio traci
         output:
             otlp:
                 endpoint:
-                    value: https://{ENVIRONMENT_ID}.live.dynatrace.com/api/v2/otlp
+                    valueFrom:
+                        secretKeyRef:
+                            name: dynakube
+                            namespace: ${DYNATRACE_NS}
+                            key: apiurl
+                path: v2/otlp/v1/metrics
```

Comment:
This path seems to be for metrics, not traces

Reply:
fixed